puppet/zulip/files/postgresql/process_fts_updates

```diff
@@ -104,8 +104,11 @@ if 'host' in pg_args:
     else:
         pg_args['sslmode'] = 'verify-full'
     pg_args['connect_timeout'] = '600'
-else:
+elif config_file.has_option('machine', 'deploy_type'):  # We are on production
     pg_args['user'] = 'zulip'
+else:  # Use zulip_test db in developments setups.
+    pg_args['user'] = 'zulip_test'
+    pg_args['dbname'] = 'zulip_test'
```

Comment:
This is wrong; we want to use the `zulip` database in the development environment.  I think we should instead be looking at `DJANGO_SETTINGS_MODULE`.

Will require changes to the `os.environ['DJANGO_SETTINGS_MODULE'] = 'zproject.settings'` block above.

(BTW a design goal there is that this imports on a postgres server with no venv).

Reply:
@timabbott Do we even need to run this process at all when we're doing API or Casper tests?  Those tests use the test database, so I don't understand how they'd work with `process_fts_updates` if the latter were using the dev database.

Reply:
It depends whether we want full-text search to be broken in the Casper and API tests; if it's not running, it will be broken for any messages sent during the duration of the test, which could be pretty confusing.